class-tour.php

```diff
@@ -151,15 +151,18 @@ public static function rest_api_init() {
 						'success' => true,
 					);
 				},
+				'permission_callback' => function () {
+					return is_user_logged_in();
+				},
```

Comment:
If you just wrap a function it can be done like this:
```suggestion
				'permission_callback' => 'is_user_logged_in',
```

Reply:
Thank you!